src/EnergyPlus/OutputProcessor.hh

```diff
@@ -904,53 +949,36 @@ namespace OutputProcessor {
-                         std::string const &KeyedValue,             // Associated Key for this variable
-                         Optional_string_const ReportFreq = _,      // Internal use -- causes reporting at this freqency
-                         Optional_int_const indexGroupKey = _       // Group identifier for SQL output
-);
-
-void SetupOutputVariable(std::string const &VariableName,           // String Name of variable
-                         OutputProcessor::Unit const &VariableUnit, // Actual units corresponding to the actual variable
-                         Real64 &ActualVariable,                    // Actual Variable, used to set up pointer
-                         std::string const &TimeStepTypeKey,        // Zone, HeatBalance=1, HVAC, System, Plant=2
-                         std::string const &VariableTypeKey,        // State, Average=1, NonState, Sum=2
-                         int const KeyedValue,                      // Associated Key for this variable
-                         Optional_string_const ReportFreq = _,      // Internal use -- causes reporting at this freqency
-                         Optional_string_const ResourceTypeKey = _, // Meter Resource Type (Electricity, Gas, etc)
-                         Optional_string_const EndUseKey = _,       // Meter End Use Key (Lights, Heating, Cooling, etc)
-                         Optional_string_const EndUseSubKey = _,    // Meter End Use Sub Key (General Lights, Task Lights, etc)
-                         Optional_string_const GroupKey = _,        // Meter Super Group Key (Building, System, Plant)
-                         Optional_string_const ZoneKey = _,         // Meter Zone Key (zone name)
-                         Optional_int_const ZoneMult = _,           // Zone Multiplier, defaults to 1
-                         Optional_int_const ZoneListMult = _,       // Zone List Multiplier, defaults to 1
-                         Optional_int_const indexGroupKey = _       // Group identifier for SQL output
+                         std::string const &VariableName,                       // String Name of variable
+                         OutputProcessor::Unit const &VariableUnit,             // Actual units corresponding to the actual variable
+                         int &ActualVariable,                                   // Actual Variable, used to set up pointer
+                         OutputProcessor::eTimeStepType const &TimeStepTypeKey, // Zone, HeatBalance=1, HVAC, System, Plant=2
+                         OutputProcessor::eVariableType const &VariableTypeKey, // State, Average=1, NonState, Sum=2
+                         std::string const &KeyedValue,                         // Associated Key for this variable
+                         Optional_string_const ReportFreq = _,                  // Internal use -- causes reporting at this freqency
+                         Optional_int_const indexGroupKey = _                   // Group identifier for SQL output
```

Comment:
Converted 2 SetupOutputVariable calls to the new enum, and the 3rd SetupOutputVariable overload was not being used so it was deleted.

Reply:
Do all of these things really need to be passed by reference? And can KeyedValue be passed by `string_view`?

Reply:
I doubt they need to be passed by reference -- definitely not the new enumeration arguments anyway.  I'll have to see how KeyedValue is used, but if it can be a string view this will be a great demo of it.  

Luckily now we only have 2 SetupOutputVariable calls to clean up instead of 3.